keyboards/dp3000/info.json

```diff
@@ -0,0 +1,99 @@
+{
+    "manufacturer": "depermana12",
+    "keyboard_name": "dp3000",
+    "maintainer": "depermana12",
+    "diode_direction": "COL2ROW",
+    "development_board": "promicro",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "encoder": true,
+        "encoder_map": true,
```

Comment:
Encoder map should only be enabled at keymap level. 
```suggestion
```

Reply:
could you please elaborate how to enabled `encoder_map` at keymap level